### PR TITLE
Mayowa

### PR DESCRIPTION
**Pull request by MAYII-XCHANGE**
Done with the work

### CODE REVIEW

File: assets/MAYOWA DOM PROJECT 1.png
Comment (Sesughter01) [inline assets/MAYOWA DOM PROJECT 1.png:1]: Please the name of the html file this DOM is for

File: class_eight/index.html
Comment (Sesughter01) [inline class_eight/index.html:1]: Your scope of work did not include adding a header element , your team was only required to work on the section below the header. Remove the header element and its contents then stage commit , push and create a new pull request with the changes made. thank you. well done.

File: class_two/index.css
Comment (Sesughter01) [inline class_two/index.css:1]: I will view this work and revert shortly. Good work

File: class_two/index.html
Comment (Sesughter01) [inline class_two/index.html:1]: I will view and revert shortly

File: class_two/news-homepage-main/index.html
Comment (Sesughter01) [inline class_two/news-homepage-main/index.html:1]: I will view and revert shortly

**Review by Sesughter01 — CHANGES_REQUESTED** [review]
Well doe. Make the changes to class_eight as required and indicate the name of the html file your DOM is for. I will view your news-homepage-main project and score you.

**Comment by MAYII-XCHANGE** [comment]
JAVASCRIPT CLASS WORK

**Pull request closed without merging**